vagrant/src/test/java/org/jclouds/vagrant/compute/WindowsLiveTest.java

```diff
@@ -103,4 +109,11 @@ public void testBoxConfig() {
       assertEquals(boxConfig.getStringKey(".vm.communicator"), Optional.of("winrm"));
       assertEquals(boxConfig.getKey(VagrantConstants.KEY_VM_GUEST), Optional.of(VagrantConstants.VM_GUEST_WINDOWS));
    }
+
+   private void skipIfImageNotPresent() {
+      Image image = view.getComputeService().getImage(getImageId());
```

Comment:
The `getImage` call is not cached (as opposed to the `listImages` one). Better get the image in a `@Before` method just once?